scripts/convert_build_commands.py

```diff
@@ -43,6 +45,7 @@ def convert_entries(entries, out_dir=None):
             elif arg == "-o":
                 ei.output = next(arg_iter)
                 ei.new_args.append(arg)
+                assert ei.output is not None
```

Comment:
`# for mpypy` again (see above comment).

Reply:
This is a correctness condition.